cura/API/Account.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, application: "CuraApplication", parent = None) -> None:
             OAUTH_SERVER_URL= self._oauth_root,
             CALLBACK_PORT=self._callback_port,
             CALLBACK_URL="http://localhost:{}/callback".format(self._callback_port),
-            CLIENT_ID="um---------------ultimaker_cura_drive_plugin",
+            CLIENT_ID="um----------------------------ultimaker_cura",
```

Comment:
@ChrisTerBeke Do you know what's this `CLIENT_ID` for? Why there's so many dashes in it?

Reply:
Our OAuth2 server specifies an application client identifier of 44 characters should be used. This is the exact ID that is registered in the authorization server for Cura.